code/game/objects/structures/girders.dm

```diff
@@ -167,7 +167,7 @@
 						qdel(src)
 					return
 
-		if(S.sheettype)
+		if(S.sheettype && S.sheettype != "runed")
```

Comment:
ewwwww

Reply:
All of girder code is EWWWW I'm just following form.